### ISSUE TITLE
saving search preferences

### ISSUE THREAD

**Issue by brente78** [Film-Aficionado/filmaf.com#2]
Search preferences only save for your current session. When you leave and then return to the site you have to set them again for the new session.

**Comment by drdouglasghd** [comment]
I suppose this used to work this way?

**Comment by brente78** [comment]
Yes, it used to work so that every time I logged in it pulled up my saved search preferences. Now I have to set my preferred search preferences every day.
